src/components/EditEvent/EventInformation.tsx

```
@@ -40,19 +62,102 @@
+            applicationPeriod: {
+              startDate: parseDate(selectedRange?.from),
+              endDate: parseDate(selectedRange?.to),
+            },
+            startAt: parseDate(selectedEventDate),
+            mainEventMaxApplicantCount: mainEventLimitEnabled
+              ? parseInt(mainEventMaxCount) || 0
+              : null,
+            afterPartyMaxApplicantCount: afterPartyLimitEnabled
+              ? parseInt(afterPartyMaxCount) || 0
+              : null
+              
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)